src/components/pages/NewProject/NewProject.js

```diff
@@ -1,126 +1,211 @@
+// NPM import
 import React, { useState } from "react";
+import { useForm } from "react-hook-form";
 import "../../../../node_modules/bootstrap/dist/css/bootstrap.min.css";
 import "./styles/NewProject.css";
-import { useForm } from "react-hook-form";
-import { useHistory } from "react-router-dom";
+
+// Component
+import Years from "./components/Years";
+import Months from "./components/Months";
+import ProjectName from "./components/ProjectName";
+import Client from "./components/Client";
+import Description from "./components/Description";
+import Location from "./components/Location";
+import Dates from "./components/Dates";
+import MemberList from "./components/MemberList";
+
+// API route
+import projectService from "../../../Services/ProjectService";
 
 function NewProject(props) {
-  // Using the useHistory hook for pushing a new route into the history
-  const history = useHistory();
+  // ---Initialization---
 
   // Declare hooks from useForm
-  const { register, handleSubmit, newProject } = useForm();
-  const [message, setMessage] = useState(null);
+  const { register, handleSubmit, errors } = useForm();
+  const [members, setMembers] = useState([]);
+
+  // Variables for year, month and date
+  let years = [];
+  let yearsMin = 1990;
+  let yearsMax = 2020;
+  let i;
+  for (i = yearsMin; i < yearsMax + 1; i++) {
+    years.push(i);
+  }
+
+  let months = [];
+  const monthsMin = 1;
+  const monthsMax = 12;
+  let j;
+  for (j = monthsMin; j < monthsMax + 1; j++) {
+    months.push(j);
+  }
+
+  let dates = [];
+  const datesMin = 1;
+  const datesMax = 31;
+  let k;
+  for (k = datesMin; k < datesMax + 1; k++) {
+    dates.push(k);
+  }
+
+  // Get select of team members
+  const teamMember = ["ian", "alex", "amy"];
 
-  // Event to handle user adding a new project
-  const onSubmit = (user, event) => {
-    event.preventDefault();
+  // ---Event Handler---
 
-    // Placeholder for submitting a new project using an axios call to the backend application
+  // When Submit Button is Clicked
+  const onSubmit = (user, e) => {
+    const {
+      startD,
+      startMonth,
+      startYear,
+      endD,
+      endMonth,
+      endYear,
+    } = user;
+    const startDate = `${startD}-${startMonth}-${startYear}`;
+    const endDate = `${endD}-${endMonth}-${endYear}`;
+    const teamMembers = members;
+
+    const dataSend = {
+      user,
+      startDate,
+      endDate,
+      teamMembers
+    };
+    console.log(dataSend);
+
+    projectService.createProject(dataSend)
+      .then((res) => {
+        if (res.data.status === "error") {
+          throw new Error(res.data.message);
+        }
+      })
+      .catch((err) => {
+        return err;
+      });
+    //e.target.reset();
+  };
+
+  // Handler for Members
```

Comment:
Amend comment to:
`// Functions to handle user selecting and unselecting members of the project`